schema_based_on_diagram.sql

```diff
@@ -0,0 +1,79 @@
+CREATE TABLE "treatments"(
+    "id" INTEGER NOT NULL,
+    "type" VARCHAR(255) NOT NULL,
+    "name" VARCHAR(255) NOT NULL
+);
+ALTER TABLE
+    "treatments" ADD PRIMARY KEY("id");
+CREATE TABLE "medical_histories"(
+    "id" INTEGER NOT NULL,
```

Comment:
- [ ] Kindly consider adding `PRIMARY KEY` in this line as this id should be considered the primary key of this table. I can see that you have Altered this table and did the same thing on line 16 , but I think the concise and better way is to add the primary key inside the table as I mentioned how to do it.